README.md

```diff
@@ -76,18 +116,207 @@ load(
 )
 ```
 
-Then use `codechecker_test()` rule passing targets you call CodeChecker for:
+Create a `codechecker_test()` target by passing other targets you'd like CodeChecker to analyze:
+
+<!-- TODO: Consider using https://github.com/bazelbuild/stardoc to document parameters -->
+```python
+codechecker_test(
+    name = "your_codechecker_rule_name",
+    targets = [
+        "your_target",
+    ],
+)
+```
+
+Then invoke bazel:
+
+```bash
+bazel build ://your_codechecker_rule_name
```

Comment:
The idea behind codechecker_test() rules is the possibility to run CodeChecker as Bazel test: `bazel test ...` and/or `bazel test ... --test_tag_filter=codechecker` meaning that CodeChecker can be a part of verification process. Running as build rule is also possible though but has less practical meaning.